src/components/ProgressBar.css

```diff
@@ -0,0 +1,62 @@
+.progressBarContainer {
+    text-align: center;
+    position: relative;
+    margin-top: 30px;
+}
+
+/*progressbar*/
```

Comment:
This is CSS Voodoo. I am not worthy to be reviewing this dark magic. But it looks soo good.